Update README to reflect IDE requirements

After cloning and importing a project I'm met with following error:

```

FAILURE: Build failed with an exception.

* What went wrong:
This version of Android Studio cannot open this project, please retry with Android Studio 3.6 or newer.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 1m 3s
This version of Android Studio cannot open this project, please retry with Android Studio 3.6 or newer.

```

While README states:

> 
> ### Android Studio IDE setup
> 
> Plaid requires Android Studio version 3.4 or higher.

While the issue it itself is no big deal it does cause a bit of inconvenience.

## Steps to reproduce:
### Inside android studio
File -> New -> Project from version control -> Git -> paste into URL field "https://github.com/android/plaid.git" -> press Clone and open project afterwards.

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ Changes to the styles, themes, icons, animations, transitions or any other UI el
 
 ### Android Studio IDE setup
 
-Plaid requires Android Studio version 3.4 or higher.
+Plaid requires Android Studio version 3.6 or higher.
 
 Plaid uses [ktlint](https://ktlint.github.io/) to enforce Kotlin coding styles.
 Here's how to configure it for use with Android Studio (instructions adapted

